src/rgw/store/daos/README.md

```diff
@@ -0,0 +1,27 @@
+# DAOS
+Standalone Rados Gateway (RGW) on [DAOS](http://daos.io/) (Experimental)
+
+
+## CMake Option
+Add below cmake option
+
+    -DWITH_RADOSGW_DAOS=ON 
+
+
+## Build
+
+    cd build
+    ninja [vstart]
+
+
+## Running Test cluster
+Edit ceph.conf to add below option
+
+    [client]
+        rgw backend store = daos
+
+Restart vstart cluster or just RGW server
+
+    [..] RGW=1 ../src/vstart.sh -d
+
+The above configuration brings up RGW server on DAOS and creates testid user to be used for s3 operations.
```

Comment:
nit: I dont see in the code where testid is being created by default for sal_daos. The doc here might need correction.